README.md

```diff
@@ -48,6 +49,7 @@ targets:
 
 features:
   enable_source_link: false
+  enable_code_block: false
```

Comment:
We should add a screenshot or markdown sample showing what these different features do.